pipdecode.js

```diff
@@ -0,0 +1,32 @@
+require('buffertools').extend();
```

Comment:
If you like, I am happy to explore a few different directions.
We could try a few different angles on the parser and the cli.